docs/release_notes/v1.0.0-rc.3.md

```diff
@@ -0,0 +1,518 @@
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+
+kubectl create namespace dapr-system
+
+helm install dapr dapr/dapr --version 1.0.0-rc.3 --namespace dapr-system --wait
+```
+
+Alternatively, you can use the latest version of CLI:
+
+```
+dapr init --runtime-version=1.0.0-rc.3 -k
+```
+
+##### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ kubectl get pods -n dapr-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+dapr-dashboard-85cc6c9f7d-hzk69         1/1     Running   0          13m
+dapr-operator-84985cb985-h2zdv          1/1     Running   0          13m
+dapr-placement-server-0                 1/1     Running   0          13m
+dapr-sentry-5974c7cb6d-9bnqd            1/1     Running   0          13m
+dapr-sidecar-injector-5b5bb9454-sfmmx   1/1     Running   0          13m
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+- **RESOLVED** Make required the version field on Component YAML [2260](https://github.com/dapr/dapr/issues/2260)
+
+- **FIX** Empty etag treated as no etag. [2320](https://github.com/dapr/dapr/issues/2320)
+
+- **Change** response handling for cross-protocol invocation. [2342](https://github.com/dapr/dapr/issues/2342)
+
+- **RESOLVED** Return HTTP 409 Conflict when ETag doesn't match [2619](https://github.com/dapr/dapr/issues/2619)
+
+### .NET SDK
+
+- **RESOLVED** Remove Query all statenames from IActorStateManager because it does not behave as expected [53](https://github.com/dapr/dotnet-sdk/issues/53)
+
+- **RESOLVED** Prefer IDictionary<TKey, TValue> over Dictionary<TKey, TValue> in public APIs [395](https://github.com/dapr/dotnet-sdk/issues/395)
+
+- **RESOLVED** TrySaveStateAsync and TryDeleteStateAsync should only handle "rejections" based on ETag, not all RPC exceptions [426](https://github.com/dapr/dotnet-sdk/issues/426)
+
+- **RESOLVED** Actors HTTP handlers don't work with some common frontend server patterns [434](https://github.com/dapr/dotnet-sdk/issues/434)
+
+- **RESOLVED** Have consistent API names for .NET SDK [500](https://github.com/dapr/dotnet-sdk/issues/500)
+
+- **RESOLVED** Throw DaprException for dapr errors [516](https://github.com/dapr/dotnet-sdk/issues/516)
+
+- **RESOLVED** Use Task instead of ValueTask [517](https://github.com/dapr/dotnet-sdk/issues/517)
+
+- **RESOLVED** Reimagining service invocation for dotnet-sdk [526](https://github.com/dapr/dotnet-sdk/issues/526)
+
+- **RESOLVED** Simplifying Service Invocation API surface [549](https://github.com/dapr/dotnet-sdk/issues/549)
+
+- **RESOLVED** Expose Dapr API token to users as a settable value [551](https://github.com/dapr/dotnet-sdk/issues/551)
+
+- **FIX** DaprClient now implements IDisposable so that networking resources can be reclaimed deterministically. [559](https://github.com/dapr/dotnet-sdk/issues/559)
+
+### Go SDK
+
+- **RESOLVED** Remove unstable dependencies. [136](https://github.com/dapr/go-sdk/issues/136)
```

Comment:
Is this change actually in the release? I don't see a corresponding commit/PR and it's still in To Do in Milestone 3.

Reply:
It's been closed. I'll remove it from the hack/md.